src/module/lighthouse/index.ts

```diff
@@ -16,22 +17,27 @@ import { CommandOptions } from '@/typings/utils/command';
 import defaultConfig from './default-config';
 import parseReport from './output';
 
+// Default config options
+const defaults = {
+  output: ['json'],
+};
+
 const lighthouseRunner = async (
   url: string,
   userOptions: Options,
   commandOptions: CommandOptions,
   config: LighthouseConfig = Config.get('configs.lighthouse', defaultConfig),
 ): Promise<Report> => {
-  // Default config options
-  const defaults = {
-    output: ['json'],
-    maxWaitForFcp: 60 * 1000,
-  };
-
   // Build options but let users change the defaults if needed
   const options = {
-    ...defaults,
-    ...userOptions,
+    ...deepmerge(defaults, userOptions),
+    ...Config.getObject([
+      {
+        config: 'configs.lighthouse.maxWaitForFcp',
```

Comment:
is there a way to use `configs.lighthouse.*` sans thresholds and any other config we use for gimbal?

Reply:
I think you are preemptively trying to solve something that isn't a problem and may not ever be a problem. There isn't an issue with passing `thresholds` to lighthouse, it just simply won't every check or use it. Could they add support for that? Sure but we already pass the `configs.lighthouse` blindly to lighthouse too.

You could get `configs.lighthouse` and that would be an object. We could add a flatten option where it takes that lighthouse object and throws it up a level. We could also have a removeProps that takes an array of strings to `delete` them.

Reply:
There, just added a way to get the `configs.lighthouse` object, flatten it up to the returned object and also remove certain props (`threshold` and `outputHtml` for lighthouse).

Reply:
I might be, but I already wondered about blacklisting lighthouse audits. This approach might be the solution to things like that